src/nvim/api/private/helpers.c

```diff
@@ -1069,8 +1083,18 @@ ArrayOf(Dictionary) keymap_array(String mode, buf_T *buf)
       // Check for correct mode
       if (int_mode & current_maphash->m_mode) {
         mapblock_fill_dict(dict, current_maphash, buffer_value, false);
-        ADD(mappings, vim_to_object((typval_T[]) { { .v_type = VAR_DICT, .vval.v_dict = dict } }));
-
+        Object api_dict = vim_to_object((typval_T[]) { { .v_type = VAR_DICT,
+                                                         .vval.v_dict = dict } });
+        // if (from_lua) {
+        //   Dictionary d = api_dict.data.dictionary;
+        //   for (size_t j = 0; j < d.size; j++) {
+        //     if (strequal("luaref", d.items[j].key.data)) {
+        //       PUT(d, "func", LUAREF_OBJ((LuaRef)d.items[j].value.data.integer));
```

Comment:
This doesn't work if I try to put anything in the dict (I tried even putting a bool) I get a `string length overflow` error from `luajit` . Any idea why ?